### ISSUE TITLE
Undefined symbol: PyUnicodeUCS2_AsWideChar

### ISSUE THREAD

**Issue by nokome** [stencila/stencila#107]
Using Python build from Travis CI gave this error on one machine:

```
    import stencila
  File "/srv/stencila/curator/env/local/lib/python2.7/site-packages/stencila/__init__.py", line 5, in <module>
    from stencila.component import Component, grab
  File "/srv/stencila/curator/env/local/lib/python2.7/site-packages/stencila/component.py", line 1, in <module>
    import stencila.extension as extension
ImportError: /srv/stencila/curator/env/local/lib/python2.7/site-packages/stencila/extension.so: undefined symbol: PyUnicodeUCS2_AsWideChar
```

Test on other machines

**Comment by nokome** [comment]
Also happening with `stencila/ubuntu-14.04-py-2.7` images build on Travis CI:

```
Traceback (most recent call last):
  File "/usr/local/bin/stencila-session", line 6, in <module>
    import stencila
  File "/usr/local/lib/python2.7/dist-packages/stencila/__init__.py", line 5, in <module>
    from stencila.extension import serve
ImportError: /usr/local/lib/python2.7/dist-packages/stencila/extension.so: undefined symbol: PyUnicodeUCS2_AsWideChar
```

**Comment by nokome** [comment]
Fixed with 5c1f895. There are several issues over on travis-ci/travis-ci related to this. The most relevant being travis-ci/travis-ci#5107 but a solution in travis-ci/travis-ci#4948
